ngx-fudis/package.json

```diff
@@ -23,53 +23,56 @@
     "@angular/common": "^17.3.10",
     "@angular/compiler": "^17.3.10",
     "@angular/core": "^17.3.10",
-    "@angular/forms": "^17.3.10",
+    "@angular/forms": "^17.3.11",
     "@angular/material": "^17.3.10",
     "@angular/platform-browser": "^17.3.10",
-    "@angular/platform-browser-dynamic": "^17.3.10",
-    "@angular/router": "^17.3.10",
+    "@angular/platform-browser-dynamic": "^17.3.11",
+    "@angular/router": "^17.3.11",
     "eslint-config-prettier": "^9.1.0",
     "rxjs": "^7.8.1",
-    "tslib": "^2.6.2",
-    "zone.js": "^0.14.6"
+    "tslib": "^2.6.3",
+    "zone.js": "^0.14.7"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "^17.3.8",
-    "@angular-eslint/eslint-plugin": "^17.5.1",
-    "@angular-eslint/eslint-plugin-template": "^17.5.1",
-    "@angular-eslint/template-parser": "^17.5.1",
+    "@angular-eslint/eslint-plugin": "^17.5.2",
+    "@angular-eslint/eslint-plugin-template": "^17.5.2",
+    "@angular-eslint/template-parser": "^17.5.2",
     "@angular/cli": "^17.3.8",
-    "@angular/compiler-cli": "^17.3.10",
-    "@babel/core": "^7.24.6",
+    "@angular/compiler-cli": "^17.3.11",
+    "@babel/core": "^7.24.7",
     "@compodoc/compodoc": "^1.1.25",
     "@ngneat/transloco": "^6.0.4",
-    "@storybook/addon-a11y": "^7.6.19",
+    "@storybook/addon-a11y": "^7.6.20",
     "@storybook/addon-actions": "^7.6.19",
-    "@storybook/addon-essentials": "^7.6.19",
-    "@storybook/addon-interactions": "^7.6.19",
-    "@storybook/addon-links": "^7.6.19",
-    "@storybook/addon-mdx-gfm": "^7.6.19",
-    "@storybook/angular": "^7.6.19",
+    "@storybook/addon-essentials": "^7.6.20",
+    "@storybook/addon-interactions": "^7.6.20",
+    "@storybook/addon-links": "^7.6.20",
+    "@storybook/addon-mdx-gfm": "^7.6.20",
+    "@storybook/angular": "^7.6.20",
     "@types/jest": "^29.5.12",
-    "@types/node": "^20.11.21",
-    "@typescript-eslint/eslint-plugin": "^7.10.0",
-    "@typescript-eslint/parser": "^7.10.0",
+    "@types/node": "^20.14.10",
+    "@typescript-eslint/eslint-plugin": "^7.16.0",
+    "@typescript-eslint/parser": "^7.16.0",
     "babel-loader": "^9.1.3",
     "eslint": "^8.57.0",
     "jest": "^29.7.0",
-    "jest-preset-angular": "^14.1.0",
-    "ng-mocks": "^14.12.2",
+    "jest-preset-angular": "^14.1.1",
+    "ng-mocks": "^14.13.0",
     "ng-packagr": "^17.3.0",
-    "prettier": "^3.2.5",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "prettier": "^3.3.2",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
     "semver-parser": "^4.1.6",
-    "storybook": "^7.6.19",
-    "stylelint": "^16.6.0",
+    "storybook": "^7.6.20",
+    "stylelint": "^16.6.1",
     "stylelint-config-concentric-order": "^5.2.0",
     "stylelint-config-sass-guidelines": "^11.1.0",
     "stylelint-config-standard-scss": "^13.1.0",
     "ts-node": "^10.9.2",
-    "typescript": "^5.2.2"
+    "typescript": "^5.4.5"
```

Comment:
I would also update `tslib` and `typescript` in `devDependencies` of `ngx-fudis/projects/ngx-fudis/package.json` to match the versions specified here. Probably no biggie, but in edge cases might avoid some weird behavior while developing. And anyway [`devDependencies` changes are not considered breaking changes](https://softwareengineering.stackexchange.com/questions/440018/semantic-versioning-on-a-dev-dependencies-change) so should be trivial to keep them in sync with this file.